### ISSUE TITLE
Extended build fails with new Xcode 14.1 on MacOS

### ISSUE THREAD

**Issue by onedrawingperday** [gohugoio/hugo#10491]
### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version

hugo v0.107.0-dc44bca963b0035818d4521b73c39af63db3a230+extended darwin/arm64 BuildDate=2022-12-02T11:13:34Z

</pre>

### Does this issue reproduce with the latest release?
No

To reproduce pull the latest changes from Hugo master branch and then try compiling on macOS with:
`env GOOS=darwin GOARCH=arm64 CGO_ENABLED=1 go build -tags extended -o $HOME/bin` or similar.

The compiling fails with the following console message:

```
go: downloading github.com/bep/godartsass v0.15.0
go: downloading google.golang.org/protobuf v1.28.1
# github.com/bep/golibsass/internal/libsass
In file included from json.cpp:2:
../../../go/pkg/mod/github.com/bep/golibsass@v1.1.0/internal/libsass/../../libsass_src/src/json.cpp:1289:3: warning: 'sprintf' is deprecated: This function is provided for compatibility reasons only.  Due to security concerns inherent in the design of sprintf(3), it is highly recommended that you use snprintf(3) instead. [-Wdeprecated-declarations]
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/stdio.h:188:1: note: 'sprintf' has been explicitly marked deprecated here
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/cdefs.h:215:48: note: expanded from macro '__deprecated_msg'

```

**Comment by onedrawingperday** [comment]
For now I have set Xcode to “Continue building after errors“ :.

![](https://i.stack.imgur.com/VfAck.png)

The above deprecation shipped with the recently released Xcode 14.1

**Comment by bep** [comment]
Are you sure that the build actually fails (what is the output of `echo $?` after a build)? It's rather odd that they then call it a compiler _warning_.

**Comment by onedrawingperday** [comment]
Hello,

I am seeing empty output when I execute `echo` right after the above stream of Xcode errors. 

Also the timestamp of the Hugo build remains unchanged.

For example tried to build from source just now [@ 14:10 pm EET+2] the timestamp in the output of `hugo version` shows 11:13 EET+2:
```
hugo v0.107.0-dc44bca963b0035818d4521b73c39af63db3a230+extended darwin/arm64 BuildDate=2022-12-02T11:13:34Z
``` 



P.S. Also the above post I made about tweaking Xcode settings has no effect. I got confused because I succesully built the plain Hugo version.

It is not possible to build Hugo Extended as of now in macOS Ventura with Xcode 14.1, so I guess I will start using the release binaries.
